handlers/aws/utils.py

```diff
@@ -313,8 +313,11 @@ def get_trigger_type_and_config_source(event: dict[str, Any]) -> tuple[str, str]
                 and "eventSource" in body["Records"][0]
             ):
                 event_source = body["Records"][0]["eventSource"]
+                if event_source not in _available_triggers:
+                    raise Exception
```

Comment:
I would add an exception message of some kind here

Reply:
I added a message only where the exceptions will bubble up to the function callar